### DISCUSSION TITLE
it randomly stops working

### DISCUSSION THREAD

**Discussion by BlackBullNetwork** [mishl-dev/Discord-AI-Chatbot#146] [Q&A]
when i run the bot it works fine but after a few hours it stops working if i try to chat with it or even if i try /imagine it wont work anymore is there a limitation on how much you can do per day ??

**Comment by mishl-dev** [comment]
> when i run the bot it works fine but after a few hours it stops working if i try to chat with it or even if i try /imagine it wont work anymore is there a limitation on how much you can do per day ??


You have to continuously run the bot. Discord doesn't host bots themselves.

**Comment by BlackBullNetwork** [comment]
i am running the bot on a local server the bot doesnt stop just the ai part stops
its at random moments that it just stops and then the next day it works again 
do i need to use my own api keys somewhere or something??

**Reply by mishl-dev** [reply]
> i am running the bot on a local server the bot doesnt stop just the ai part stops its at random moments that it just stops and then the next day it works again do i need to use my own api keys somewhere or something??

Idk when did you cloned the repo

**Reply by BlackBullNetwork** [reply]
yesteday i cloned the repo

**Comment by BlackBullNetwork** [comment]
yesteday i cloned the repo

**Reply by mishl-dev** [reply]
alright its just need to update api url

**Reply by BlackBullNetwork** [reply]
Okey how and where cause it should keep working i assume where do i get the correct api url cause rn i need to keep updating the project

**Comment by BlackBullNetwork** [comment]
Okey how and where cause it should keep working i assume where do i get the correct api url cause rn i need to keep updating the project
